amy/extforms/forms.py

```diff
@@ -177,13 +177,14 @@ def set_consent_fields(self, terms: Iterable[Term]) -> None:
 
     def create_consent_field(self, term: Term) -> forms.ChoiceField:
         options = [(opt.pk, option_display_value(opt)) for opt in term.options]
+        label = term.training_request_content or term.content
```

Comment:
I think there is a similar function in another form (ConsentForm?). Should we put this line over there, too?

Reply:
Additionally you marked the field `training_request_content` as nullable. This means it can have 2 "empty" values: `None` and `""`.

For `CharFields` I always prefer to have 1 empty value (`""`) and only accept nullable if there's a good reason for it.

I think here it would be better to keep the field non-nullable with a default value of `""`, so that we are handling it correctly in forms and there are no hidden surprises.